### PR TITLE
Feature/dashboard screen

### PR DESCRIPTION
**Pull request by Humnaa**
https://trello.com/c/kv1Uuksz/19-create-apps-dashboard

### CODE REVIEW

**Comment by abdulhaqshah** [comment]
@Humnaa nice work. But there are still some changes we have to do. 

First the page is not properly aligned.
Background color is different
Tutorial part is missing
Header bar lots of things are missing like user profile
Font size are different 

Dashboard should be look like this
image

File: src/containers/Dashboard/Card.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div className="courses-card">
+        <div
+          className="card-div shadow-lg"
+          className="card"
+          style={{ width: "22rem", height: "30rem" }}
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Card.js]: Do not use inline styling, every style should be defined in the CSS of that component

File: src/containers/Dashboard/Card.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        >
+          <div className="card-body">
+            <p className="text-muted">{this.props.title}</p>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Card.js]: Not a good approach use div and try to make this with css. Using direct lines and p tags case problems. Handle it with div and css

File: src/containers/Dashboard/Card.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+import "./Card.scss";
+class Card extends Component {
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Card.js:4]: add one line space between import and class

File: src/containers/Dashboard/ContentContainer.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+import "./ContentContainer.scss";
+import Card from "./Card";
+import Footer from "../../components/Footer";
+import Tutorial from "../Dashboard/Tutorial";
+
+class ContentContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js:9]: alway use a one line space between functions

File: src/containers/Dashboard/ContentContainer.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    const link1 = "View Tutorial";
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js]: Variable naming convention should be proper never use 1 2 3 or a1 b2 c3 button 2  kind of names

File: src/containers/Dashboard/ContentContainer.js [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+          </div>
+          <div className="ml-5 font-weight-bold">
+            <h5 className="heading">Dashboard</h5>
+          </div>
+        </header>
+        <div className="main-container">
+          <h5 className="ml-5 mb-4 font-weight-bold">Your Skills</h5>
+
+          <div className="interview-card1 shadow-lg mr-4 ml-5">
+            <div className="card" style={{ width: "35rem", height: "15rem" }}>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js]: again inline styling

File: src/containers/Dashboard/ContentContainer.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+                  compaanies to help you prepare for your upcoming interviews.
+                </p>
+                <a href="#" className="btn shadow-lg btn-success btn-lg ">
+                  View
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="interview-card2 shadow ml-5 ">
+            <div className="card" style={{ width: "35rem", height: "15rem" }}>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js]: inline styling

File: src/containers/Dashboard/ContentContainer.js [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+            >
+              <div className="card-body">
+                <p className="text-muted">PROBLEM SOLVING</p>
+                <h2 className="card-title">Problem Solving</h2>
+                <div className="progress-div">
+                  <div class="progress">
+                    <div
+                      class="progress-bar"
+                      role="progressbar"
+                      style={{ width: " 25%" }}
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js]: same

File: src/containers/Dashboard/ContentContainer.js [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+                  Resend Confirmation Email
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="interview-card3 shadow-lg ml-5 mt-5">
+            <div
+              className="card-div"
+              className="card"
+              style={{ width: "35rem", height: "15rem" }}
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/ContentContainer.js]: same remove inline styling form all the code

File: src/containers/Dashboard/Dashboard.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import Navbar from "./Navbar";
+import ContentContainer from "../Dashboard/ContentContainer";
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div>
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Dashboard.js:8]: If you are using div just because to make this in one tag you can use Fragment of react

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/containers/Dashboard/Card.js (hunk 2 above)
Comment (Humnaa) [reply src/containers/Dashboard/Card.js]: I have used div instead of p tags

File: src/containers/Dashboard/Card.js (hunk 3 above)
Comment (Humnaa) [reply src/containers/Dashboard/Card.js:4]: There is one line space between import and class in every file

File: src/containers/Dashboard/ContentContainer.js (hunk 4 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js:9]: I have used one line space between functions

File: src/containers/Dashboard/ContentContainer.js (hunk 5 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js]: I have removed this kind of variables and give them suitable names

File: src/containers/Dashboard/ContentContainer.js (hunk 6 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js]: Removed inline styling

File: src/containers/Dashboard/ContentContainer.js (hunk 7 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js]: Removed Inline styling

File: src/containers/Dashboard/ContentContainer.js (hunk 8 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js]: removed

File: src/containers/Dashboard/ContentContainer.js (hunk 9 above)
Comment (Humnaa) [reply src/containers/Dashboard/ContentContainer.js]: I have removed inline styling from entire code

File: src/containers/Dashboard/Dashboard.js (hunk 10 above)
Comment (Humnaa) [reply src/containers/Dashboard/Dashboard.js:8]: I have used Fragments instead of div tag

File: src/containers/Dashboard/Courses.js [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+                    <Card title="LANGUAGES" />
+                  </div>
+                  <div className="specialized-skills shadow-lg ml-5 mr-5 mb-5">
+                    <Card title="SPECIALIZED SKILLS " />
+                  </div>
+                </div>
+              </div>
+            </Fragment>
+        )
+
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Courses.js]: remove line

File: src/containers/Dashboard/Courses.js [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+                  <div className="specialized-skills shadow-lg ml-5 mr-5 mb-5">
+                    <Card title="SPECIALIZED SKILLS " />
+                  </div>
+                </div>
+              </div>
+            </Fragment>
+        )
+                
+    }
+
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Courses.js]: remove line

File: src/containers/Dashboard/Tutorial.js [hunk 13]
```diff
@@ -0,0 +1,104 @@
+import React, { Component, Fragment } from "react";
+import "./Tutorial.scss";
+class Card extends Component {
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Tutorial.js:3]: space missing

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: src/containers/Dashboard/Courses.js (hunk 11 above)
Comment (Humnaa) [reply src/containers/Dashboard/Courses.js]: removed extra lines

File: src/containers/Dashboard/Courses.js (hunk 12 above)
Comment (Humnaa) [reply src/containers/Dashboard/Courses.js]: removed

File: src/containers/Dashboard/Tutorial.js (hunk 13 above)
Comment (Humnaa) [reply src/containers/Dashboard/Tutorial.js:3]: add space between import and class

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**